### PR TITLE
results

### PR DESCRIPTION
**Pull request by khadmiali**

### CODE REVIEW

File: report/sections/results.tex [hunk 1]
```diff
@@ -1 +1,37 @@
...
+    \label{eq7}
+\end{equation}
+By looking at the z-transform table, the equation \ref{eq7} yields to
+\begin{equation}
+    C(z)=\frac{z-1}{z}\bigg( k_p \frac{1}{1-z^{-1}}+k_I \frac{T z^{-1}}{(1-z^{-1})^2}+k_D\bigg)
+\end{equation}
+And finally the PID controller in discrete time domain is presented as
+\begin{equation}
+    C(z)= k_P + k_I \frac{1}{z-1} + k_D \frac{z-1}{z}
+\end{equation}\\
```
Comment (tmplt) [inline report/sections/results.tex]: Don't line-break (`\\`).

File: report/sections/results.tex [hunk 2]
```diff
@@ -1 +1,37 @@
 \section{Results}
+\section{Discretization of PID controller}
```
Comment (tmplt) [inline report/sections/results.tex]: Shouldn't this be in the implementation section or somewhere similar? I figure we would talk about the results of the project in this section.

File: report/sections/results.tex [hunk 3]
```diff
@@ -1 +1,37 @@
...
+    \begin{split}
+        K_P & = 1.35 \\
+        K_I & = 6 \\
+        K_D & = 0.08 \\
+    \end{split}
+\end{equation}
+The simulation is made in discrete time domain with a sampling frequency 10 Hz.
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.4\textwidth]{{sections/assets/pid_motor}
```
Comment (tmplt) [inline report/sections/results.tex]: This path is invalid, and there is an opening bracket too many.

**Review by tmplt — CHANGES_REQUESTED** [review]

File: report/sections/results.tex (hunk 2 above)
Comment (khadmiali) [reply report/sections/results.tex]: I can push all of this in the implementation section and solving the issues

**Pull request closed without merging**